Projects/Dotmim.Sync.Core/Serialization/LocalJsonSerializer.cs

```diff
@@ -475,12 +426,12 @@ private static SyncTable GetSchemaTableFromReader(JsonTextReader reader, JsonSer
 
             schemaTable = new SyncTable(tableName, schemaName);
 
-            for (int i = 0; i < includedColumns.Count; i++)
+            foreach (var includedColumn in includedColumns)
```

Comment:
Please consider reverting this change for [this](https://www.meziantou.net/fastest-way-to-enumerate-a-list-t.htm) reason.

Reply:
I also liked that this approach ensures the array lookup by index happens only once instead of 3 times.

Perhaps the compiler optimizes that away, or we could create a local variable.

Reply:
you have same result with `includedColumn  = includedColumns[i]`